packages/twenty-front/folderStructure.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "../../node_modules/eslint-plugin-project-structure/folderStructure.schema.json",
+  "regexParameters": {
+    "camelCase": "^[a-zA-Z]+([A-Z][a-z]+)+"
+  },
+  "structure": [
+    {
+      "name": "packages",
+      "children": [
+        {
+          "name": "twenty-front",
+          "children": [
+            {
+              "name": "src",
+              "children": [
+                { "name": "*", "children": [] },
+                { "name": "*" },
+                {
+                  "name": "modules",
+                  "children": [{ "ruleId": "moduleFolderRule" }]
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "rules": {
+    "moduleFolderRule": {
+      "name": "^(?!utils$|hooks$|states|types|graphql|components|effect-components|constants|validation-schemas|context$)[a-z][a-z0-9]**(?:-[a-z0-9]+)**$",
+      "folderRecursionLimit": 3,
+      "children": [
+        { "ruleId": "moduleFolderRule" },
+        { "name": "hooks", "ruleId": "hooksLeafFolderRule" },
+        { "name": "utils", "ruleId": "utilsLeafFolderRule" },
+        { "name": "states", "ruleId": "doNotCheckLeafFolderRule" },
```

Comment:
I have added this rule for now we can progressively add the rules folder by folder once we have fixed the existing warnings (we have ~150), let's do it step by step

Reply:
I've actually fixed existing warnings but I don't want to move everything now, we will activate these rules progressively